engine/src/test/java/com/datatorrent/stram/StramMiniClusterTest.java

```diff
@@ -235,6 +251,20 @@ private LogicalPlan createDAG(LogicalPlanConfiguration lpc) throws Exception
     return dag;
   }
 
+  private void addExtraTestJar(LogicalPlan dag)
+  {
+    try {
+      File tmp1 = File.createTempFile("tmp1", null);
+      File tmp2 = File.createTempFile("tmp2", null);
+      tmp1.deleteOnExit();
+      tmp2.deleteOnExit();
+      dag.addJarResource(tmp1.getAbsolutePath(), true);
+      dag.addJarResource(tmp2.getAbsolutePath(), false);
+    } catch (IOException e) {
+      LOG.error("Failed to create temporary file.", e);
```

Comment:
why not throw exception instead of just logging it? In fact why catch Exception, let calling function handle it